test/integration/test_job.py

```diff
@@ -102,6 +102,7 @@ def test_cancel_job_done(self, service):
     @run_integration_test
     def test_delete_job(self, service):
         """Test deleting a job."""
+        self.skipTest("Deleting jobs not supported on IBM Quantum Platform.")
         sub_tests = ["DONE"]
```

Comment:
Only instance admins can delete jobs so we'll also have to add a warning/proper error in `service.delete_job()`